### PR TITLE
chore: cleanup search_version

### PR DESCRIPTION
**Pull request by rebelchris**
Remove the old v1 search support as we now only support v2 on these.
We still have the logic for bookmark and reading history search

### CODE REVIEW

File: src/schema/search.ts [hunk 1]
```diff
@@ -185,11 +170,6 @@ export const typeDefs = /* GraphQL */ `
       """
       query: String!
 
-      """
-      Version of the search algorithm
-      """
-      version: Int = 1
```
Comment (capJavert) [inline src/schema/search.ts:191]: Do we want to remove it? In the future we could have multiple search versions again and then we have to return all this code.

Comment (capJavert) [reply src/schema/search.ts:191]: Instead we could just adjust what default is (if not sent), eg. now to be `2`.

Comment (rebelchris) [reply src/schema/search.ts:191]: Also open to that :) 
It might just be confusing what 2 means if there is no 1?

Comment (capJavert) [reply src/schema/search.ts]: I look at it as feed. There the old versions also phase out. So anything below current version will just get `2`.

File: src/schema/search.ts [hunk 2]
```diff
@@ -274,62 +252,43 @@ export const resolvers: IResolvers<unknown, Context> = traceResolvers({
       getSession(ctx.userId, id),
     searchPostSuggestions: async (
       source,
-      { query, version }: { query: string; version: number },
+      { query }: { query: string; version: number },
```
Comment (rebelchris) [inline src/schema/search.ts:255]: Don't use version here to avoid backward issues

File: __tests__/__snapshots__/feeds.ts.snap [hunk 3]
```diff
@@ -7,8 +7,8 @@ Object {
     "golang",
   ],
   "excludeSources": Array [
-    "b",
     "c",
+    "b",
```
Comment (capJavert) [inline __tests__/__snapshots__/feeds.ts.snap:11]: Flaky test?

File: src/schema/search.ts (hunk 2 above)
Comment (capJavert) [reply src/schema/search.ts:255]: Additionally, what does this mean for local, search no longer works because we don't have meili locally?

**Comment by rebelchris** [comment]
@capJavert Yep won't work locally indeed.
But can't recall last time we did anything that needed search output (posts wise)

Maybe something we can slowly add as mock?

File: __tests__/__snapshots__/feeds.ts.snap (hunk 3 above)
Comment (rebelchris) [reply __tests__/__snapshots__/feeds.ts.snap]: I guess so, my global snapshot update updated this one

**Review by capJavert — APPROVED** [review]
Approving, we can supply API key to the team when we get it.

**Pull request merged**